examples/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"crypto/tls"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+const (
+	// keyFileName is the name of the file where the private key will be saved
+	keyFileName = "private.pem"
+	// certFileName is the name of the file where the self-signed certificate will be saved
+	certFileName = "cert.pem"
+)
+
+func main() {
+
+	// Create a private key and a self-signed certificate
+	err := CreateSelfSignedKeyAndCertFiles(keyFileName, certFileName)
+	if err != nil {
+		log.Fatalf("Failed to create key and cert files: %v", err)
+	}
+
+	// Create a new TLS configuration
+	cfg := &tls.Config{
+		MaxVersion:               tls.VersionTLS13,
+		PreferServerCipherSuites: true,
```

Comment:
PreferServerCipherSuites is deprecated field

Reply:
removed